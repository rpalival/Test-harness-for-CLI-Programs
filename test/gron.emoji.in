{
    "message": "Hello, world! ğŸ‘‹",
    "user": {
        "name": "John Doe",
        "status": "Happy ğŸ˜Š"
    }
}