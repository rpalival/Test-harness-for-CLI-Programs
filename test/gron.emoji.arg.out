json = {};
json.message = "Hello, world! \ud83d\udc4b";
json.user = {};
json.user.name = "John Doe";
json.user.status = "Happy \ud83d\ude0a";